### PR TITLE
[SERVICE-179] Allow reordering of tab handles in tabstrip UI

### PR DESCRIPTION
**Pull request by tomrobertsOF**
- Uses the Sortable.js package to handle drag-and-drop-reordering.
- Existing drag-and-drop functionality is preserved (tearouts and moving windows between tabsets)
- Skeleton code in place to publish order updates to the tabbing service (will call API method from SERVICE-216)

### CODE REVIEW

**Comment by tomrobertsOF** [comment]
Looks like gts fix wasn't run on the tabbing repo before, so large number of whitespace changes ended up in this PR. If you ignore whitespace changes in the diff it should show the genuine changes.

File: src/provider/tabbing/TabAPIActionProcessor.ts [hunk 1]
```diff
@@ -1,246 +1,242 @@
...
+            default:
+                break;
+        }
+    }
+
+    /**
+     * Starts the drag window process & shows the drag window overlay.
+     */
+    private async _startDrag(source: TabIdentifier) {
+        console.log('Received startDrag event');
```
[the remark this answers is not in the document]
Comment (tomrobertsOF) [reply src/provider/tabbing/TabAPIActionProcessor.ts]: All of this is necessary to allow tearouts.

[the remark this answers is not in the document]
Comment (tomrobertsOF) [reply src/provider/tabbing/TabAPIActionProcessor.ts]: Unless you're talking about just the console.log. That can go.

[the remark this answers is not in the document]
Comment (pjbroadbent) [reply src/provider/tabbing/TabAPIActionProcessor.ts]: yeah, the logging.

**Comment by MichaelMCoates** [comment]
Conflicts to resolve!

File: src/provider/tabbing/TabAPIActionProcessor.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (tomrobertsOF) [reply src/provider/tabbing/TabAPIActionProcessor.ts]: Oh yea, it was. I'll pull that out tomorrow.

**Comment by tomrobertsOF** [comment]
@MichaelMCoates alot of them seem to just be the whitespace changes from gts fix conflicting with the latest merge. I'll go through them tomorrow.

**Comment by tomrobertsOF** [comment]
I'm gonna close this one off, do some merging/rebasing on my side and make a new PR for it.

**Pull request closed without merging**